src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php

```diff
@@ -159,18 +159,20 @@ private function connect()
             }
         }
 
-        $this->connection = ldap_connect($this->config['connection_string']);
+        $connectResult = ldap_connect($this->config['connection_string']);
```

Comment:
let's name it $connection

Reply:
ok
Although, as you mentioned, it does not cover the whole load.